backend/src/test/java/quarano/department/web/TrackedCaseControllerWebIntegrationTests.java

```diff
@@ -438,6 +439,23 @@ void updatingContactCaseDoesNotRequireQuarantineData() throws Exception {
 
 	}
 
+	@Test // CORE-582
+	void zipCodeNotFoundInRKIData() throws Exception {
+
+		var contactCase = createMinimalContactPayload();
+		contactCase.setZipCode("12345");
+
+		var response = mvc.perform(post("/hd/cases")
+				.content(jackson.writeValueAsString(contactCase))
+				.contentType(MediaType.APPLICATION_JSON)
+				.param("type", "contact"))
+				.andExpect(status().isBadRequest())
+				.andReturn().getResponse();
+		var document = JsonPath.parse(response.getContentAsString());
+		Object[] placeHolderForZipCode = {"12345"};
+		assertThat(document.read("zipCode", String.class)).isEqualTo(messages.getMessage("wrong.trackedPersonDto.zipCode", placeHolderForZipCode, "Message not loaded"));
```

Comment:
Please group related parts of the code logic and separate them with blank lines.
https://quarano.de/internal-test/api/docs/index.html#_blank_lines

I think the calling of the code to be tested, the parsing of the JSON and the verification are separate parts here.